cloud/aws/providerconfig/v1alpha1/types.go

```diff
@@ -160,6 +160,9 @@ type Network struct {
 	// VPC defines the cluster vpc.
 	VPC *VPC `json:"vpc"`
 
+	// InternetGatewayID is the id of the internet gateway associated with the VPC.
+	InternetGatewayID *string `json:"internetGatewayId"`
```

Comment:
i think you have to regen the api stuff, `make generate`

Reply:
Good point, will do

Reply:
Done!